quipucords/tests/scanner/acs/test_api.py

```diff
@@ -0,0 +1,42 @@
+"""Test ACS api client."""
+
+import pytest
+
+from scanner.acs.api import ACSApi, HTTPBearerAuth
+
+
+@pytest.fixture
+def mock_request_instance():
+    """Return MockRequest instance with empty headers."""
+
+    class MockRequest:
+        def __init__(self):
+            self.headers = {}
+
+    request = MockRequest()
+    return request
```

Comment:
some places in the code are doing this and I imagine this is where u took inspiration to do this.
Lemme present you a better alternative: use `mock.Mock`. If we need to follow `Request` api we could instanciate it by passing a class to follow its spec like `mock.Mock(spec=Request)`.

Reply:
Better and simpler, win win